Source/ASVideoNode.mm

```diff
@@ -360,6 +361,8 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
         if (self.playerState == ASVideoNodePlayerStateLoading && _delegateFlags.delegateVideoNodeDidRecoverFromStall) {
           [self.delegate videoNodeDidRecoverFromStall:self];
         }
+        
+        ASUnlockScope(self);
```

Comment:
As mentions in the description, `play` shouldn't be called with the lock.

Reply:
Is it worth adding a comment in code?